webtau-maven-plugin/src/main/groovy/com/twosigma/webtau/maven/WebTauMavenRunner.groovy

```diff
@@ -39,21 +39,31 @@ class WebTauMavenRunner extends AbstractMojo {
     @Parameter
     private String workingDir
 
+    @Parameter( property = "skipTests", defaultValue = "false" )
+    protected boolean skipTests
+
+    @Parameter( property = "maven.test.skip", defaultValue = "false" )
+    protected boolean skip
+
     @Override
     void execute() throws MojoExecutionException, MojoFailureException {
-        def fileSetManager = new FileSetManager()
-        def files = fileSetManager.getIncludedFiles(tests) as List
+        if (skipTests()) {
+            getLog().info("Skipping webtau tests")
+        } else {
+            def fileSetManager = new FileSetManager()
```

Comment:
will it make sense to extract `else` branch now so `if-else` is short and easy to follow?